concreate/tools.py

```diff
@@ -94,27 +95,87 @@ def load_descriptor(descriptor_path, schema_type):
     logger.debug("Loading %s descriptor from path '%s'."
                  % (schema_type,
                     descriptor_path))
-    schema_name = '%s_schema.yaml' % schema_type
+
+    if not os.path.exists(descriptor_path):
+        raise ConcreateError('Cannot find provided descriptor file')
+
+    schema_name = '%s.yaml' % schema_type
     schema_path = os.path.join(os.path.dirname(__file__),
                                'schema',
-                               schema_name)
-    if not os.path.exists(schema_path):
-        raise ConcreateError('Cannot locate schema for %s.' % schema_type)
-
+                               'schema.yaml')
+    specific_schema_path = os.path.join(os.path.dirname(__file__),
```

Comment:
move this bellow opening schema - so we have schema and specific_schema processing in continuous blocks